### ISSUE TITLE
Fail to delete lease

### ISSUE THREAD

**Issue by vdice** [deis/k8s-claimer#86]
The following error is thrown when a lease is deleted (or attempted):

```
Deleting Lease for ruinart -- fbd90282-6ed1-11e6-844c-02420a240109
2016/08/30 16:55:33 Error deleting. Code: 500, Body: couldn't create namespaces lister/deleter implementation  (tls: failed to find any PEM data in certificate input)
```

Indeed, when I try to lease the same cluster by name afterwards, I get handed a `409`, telling me that the lease hasn't actually been deleted/expired.

ping @jchauncey

**Comment by jchauncey** [comment]
ive never seen this before. Not sure what to make of it

**Comment by vdice** [comment]
@jchauncey I hadn't before, either -- it _seems_ to be new, perhaps as of the recent cluster name/version work?

It is occurring on all ci runs now (though doesn't fail the build) -- for example, https://ci.deis.io/job/workflow-test-pr/6067/console

**Comment by jchauncey** [comment]
alright yeah it mgiht be related then ill start investigating it

**Comment by jchauncey** [comment]
I wonder if this has something to do with the gke email we got earlier

**Comment by sgoings** [comment]
For context, that email says:

> Your project has multiple GKE clusters that are triggering a known etcd issue causing frequent master restarts.
> 
> The root cause is an issue with etcd 2 (https://github.com/kubernetes/kubernetes/issues/27307): empty directories are not garbage collected in a timely fashion. GKE has deployed a workaround fix in 1.3.5, but the aggressive namespace creation/deletion is still causing master instability. There will be a proper fix for the etcd issue in a future release, but in the meantime, if the you can adjust your workflow to avoid mass namespace creation/deletion, the clusters will be more stable.

**Comment by jchauncey** [comment]
those are our perma clusters that are not running 1.3.5 masters

**Comment by jchauncey** [comment]
im working on rebuilding them

**Comment by jchauncey** [comment]
im going to upgrade the k8s-claimer nodes tonight

**Comment by jchauncey** [comment]
fixed in #89
